static-website-gcp-yaml/Pulumi.yaml

```diff
@@ -0,0 +1,86 @@
+name: ${PROJECT}
+description: ${DESCRIPTION}
+runtime: yaml
+
+template:
+  description: A static website on Google Cloud
+  config:
+    gcp:project:
+      description: The Google Cloud project to deploy into
+    path:
+      description: The path to the folder containing the website
+      default: ./site
+    indexDocument:
+      description: The file to use for top-level pages
+      default: index.html
+    errorDocument:
+      description: The file to use for error pages
+      default: error.html
+
+configuration:
+  path:
+    type: String
+    default: ./site
+  indexDocument:
+    type: String
+    default: index.html
+  errorDocument:
+    type: String
+    default: error.html
+
+resources:
+
+  bucket:
+    type: gcp:storage:Bucket
+    properties:
+      location: US
+      website:
+        mainPageSuffix: ${indexDocument}
+        notFoundPage: ${errorDocument}
```

Comment:
Nit: The next CLI release does not use the `.append` file, so any comments we want them to see should appear in both until we land #325.

Reply:
Ah great, ok thanks @AaronFriel. Will drop these in there as well, thn.